src/main/java/net/snowflake/ingest/streaming/internal/ConfigureResponse.java

```diff
@@ -0,0 +1,61 @@
+package net.snowflake.ingest.streaming.internal;
+
+import net.snowflake.client.jdbc.internal.fasterxml.jackson.annotation.JsonProperty;
+
+class ConfigureResponse extends StreamingIngestResponse {
+  @JsonProperty("prefix")
+  private String prefix;
+
+  @JsonProperty("status_code")
+  private Long statusCode;
+
+  @JsonProperty("message")
+  private String message;
+
+  @JsonProperty("stage_location")
+  private StageMetadata stageMetadata;
```

Comment:
why not call the variable stageLocation?

Reply:
Renamed.